app/scripts/templates/settings/account_recovery/confirm_revoke.mustache

```diff
@@ -0,0 +1,14 @@
+<div id="account-recovery-confirm-revoke">
+    <section class="modal-panel">
+        <form novalidate>
+
+            <h3>{{#t}}Are you sure you want to revoke?{{/t}}</h3>
+            <p class="revoke-description">{{#t}}Revoking your account recovery key will not allow it to be used to recover your data in the event you reset your password.{{/t}}</p>
```

Comment:
I find the `will not allow it to be used` confusing and the "Are you sure you want to revoke?" makes "Revoking your account recovery key" seem redundant.

Could it be shorter/simpler?

> You will be unable to recovery your data in the event you reset your password.


Or...

> You will be unable to use your recovery key to access your data in the event you reset your password.